app/src/interfaces/group-detail/group-detail.vue

```diff
@@ -90,29 +90,30 @@ watch(validationMessages, (newVal, oldVal) => {
 <template>
 	<v-detail v-model="detailOpen" :start-open="start === 'open'" class="group-detail">
 		<template #activator="{ toggle, active }">
-			<v-divider
-				:style="{
-					'--v-divider-label-color': headerColor,
-				}"
-				:class="{ active, edited }"
-				:inline-title="false"
-				large
-				@click="toggle"
-			>
-				<template v-if="headerIcon" #icon><v-icon :name="headerIcon" class="header-icon" /></template>
-				<template v-if="field.name">
-					<span v-if="edited" v-tooltip="t('edited')" class="edit-dot"></span>
-					<span class="title">{{ field.name }}</span>
-				</template>
-				<v-icon
-					v-if="!active && validationMessages!.length > 0"
-					v-tooltip="validationMessages!.join('\n')"
-					class="warning"
-					name="error"
-					small
-				/>
-				<v-icon class="expand-icon" name="expand_more" />
-			</v-divider>
+			<button class="toggle-btn" type="button" @click="toggle">
+				<v-divider
+					:style="{
+						'--v-divider-label-color': headerColor,
+					}"
```

Comment:
I know this isn't technically your change, but can we make this a bind in the `style` block instead of inline styles here?